docker/Dockerize Flask App/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# Format: FROM    repository[:version]
+FROM       ubuntu:latest
+RUN apt-get update -y
+RUN apt-get install -y python-pip python-dev build-essential
+COPY . /app
+WORKDIR /app
+RUN pip install -r requirements.txt
```

Comment:
Copy toàn bộ source code, sau đó mới install requirements.
-> Mỗi khi update source code lại cài lại requirements, mặc dù có thể requirements không thay đổi.

Nên đổi lại thứ tự kiểu như:

```
COPY requirements /tmp/requirements
RUN pip3 install -r /tmp/requirements/base.txt
RUN pip3 install -r /tmp/requirements/production.txt

COPY . /usr/src/app
WORKDIR /usr/src/app
```